kordac/tests/ConfigurationTest.py

```diff
@@ -0,0 +1,112 @@
+        self.maxDiff = None
+
+    def test_default_processors_on_creation(self):
+        kordac = Kordac()
+        default_processors = kordac.processor_defaults()
+        self.assertEqual(kordac.kordac_extension.processors, default_processors)
+
+    def test_custom_processors_on_creation(self):
+        processors = {'comment', 'image'}
+        kordac = Kordac(processors=processors)
+        self.assertEqual(kordac.kordac_extension.processors, processors)
+
+    def test_custom_processors_after_creation(self):
+        kordac = Kordac()
+        processors = kordac.processor_defaults()
+        processors.add('example_processor')
+        kordac.update_processors(processors)
+        self.assertEqual(kordac.kordac_extension.processors, processors)
+
+    def test_unique_custom_processors(self):
+        processors = ['comment', 'comment', 'comment']
+        kordac = Kordac(processors=processors)
+        self.assertEqual(kordac.kordac_extension.processors, set(processors))
+        processors = list(kordac.processor_defaults())
+        processors.append('example_processor')
+        processors.append('example_processor')
+        processors.append('example_processor')
+        kordac.update_processors(processors)
+        self.assertTrue(kordac.kordac_extension.processors, processors)
+
+    def test_custom_templates_on_creation(self):
+        # This test doesn't do a perfect check that both templates are the same
+        # as we cannot access the raw string stored within the loaded template
+        # Instead we check if the rendered version of both is the same, however
+        # this doesn't check context variables are the same.
+        custom_templates = {
+            'image': '<img />',
+            'boxed-text': '<div class="box"></div>'
+        }
+        kordac = Kordac(html_templates=custom_templates)
+        for processor_name, template in custom_templates.items():
+            expected_template = jinja2.Template(template)
+            rendered_expected_template = expected_template.render()
+            stored_template = kordac.kordac_extension.jinja_templates[processor_name]
+            rendered_stored_template = stored_template.render()
+            self.assertEqual(rendered_expected_template, rendered_stored_template)
+
+    def test_custom_templates_after_creation(self):
+        # This test doesn't do a perfect check that both templates are the same
+        # as we cannot access the raw string stored within the loaded template
+        # Instead we check if the rendered version of both is the same, however
+        # this doesn't check context variables are the same.
+        kordac = Kordac()
+        custom_templates = {
+            'image': '<img />',
+            'boxed-text': '<div class="box"></div>'
+        }
+        for processor_name, template in custom_templates.items():
+            expected_template = jinja2.Template(template)
+            rendered_expected_template = expected_template.render()
```

Comment:
I do not think we should have these tests. They are not really testing in the spirit of a system and have some issues:
1. They essentially say we must be using `Jinja` for our templates.
2. Tests that ensure an html template has all the required variables should probably just check that `kordac` throws an exception.
3. If a test has to do the same thing the program has to do to compare outputs then its probably a bad test.

Solution:
I would prefer if these tests would use `kordac` (with some custom template) to convert a document and then compare expected outputs. If this is not currently possible to do in the system make an issue on the next milestone and comment out these tests.